DATA/FX/explosions.ini

```diff
@@ -802,6 +802,14 @@ effect = explosion_gas_mine
 radius = 200
 hull_damage = 300
 
+[explosion]
+nickname = explosion_oxygen_bw09_shatter

```

Comment:
I think "shatter" is not a very useful addition to the name here since nothing shatters really. But it's not very important since we have no proper naming convention for explosions yet.